packages/kbn-securitysolution-autocomplete/src/field_value_lists/index.tsx

```diff
@@ -36,6 +36,7 @@ interface AutocompleteFieldListsProps {
   selectedValue: string | undefined;
   allowLargeValueLists?: boolean;
   'aria-label'?: string;
+  showValueListModal: ElementType;
```

Comment:
Is there a reason that this is a prop of `AutocompleteFieldListsComponent`? The interface doesn't look very flexible (or typed correctly, since we pass very specific props to `showValueListModal`), and I don't see any data being reported back up... I would just delete lines 39, 59, and 123 👍

Reply:
That would also get rid of many of the changes on this branch 😉